cmd/translatesfx/translatesfx/otel.go

```diff
@@ -47,7 +55,14 @@ func saInfoToOtelConfig(cfg saCfgInfo) otelCfg {
 				"detectors": []string{"system", "env", "gce", "ecs", "ec2", "azure"},
 			},
 		},
-		Exporters: sfxExporter(cfg.accessToken, cfg.realm),
+		Exporters:  sfxExporter(cfg.accessToken, cfg.realm),
+		Extensions: map[string]interface{}{},
+	}
+	if len(rcReceivers) > 0 {
+		out.Receivers["receiver_creator"] = map[string]interface{}{
+			"receivers":       rcReceivers,
+			"watch_observers": []string{"k8s_observer"}, // TODO check observer type?
```

Comment:
believe there's an observers list in SA config we can use to populate this.  we'd want to flag unsupported ones.